packages/ui/src/modal/buy/BuyModal.tsx

```diff
@@ -158,6 +158,15 @@ export function BuyModal({
           steps?.filter((step) => step.items && step.items.length > 0) || []
         const lastStepItems =
           executableSteps[executableSteps.length - 1]?.items || []
+
+        const totalPurchases =
```

Comment:
Let's update this logic to look at the unique tx hashes instead of # of salesData responses since there can be duplicate sales attribute to a single transaction